main.go

```diff
@@ -1145,6 +1147,10 @@ func generateListener(listenPort int) (net.Listener, error) {
 			CipherSuites:       getCipherAliases(httpServerOptions.Ciphers),
 		}
 
+		if httpServerOptions.EnableHttp2 {
+			tlsConfig.NextProtos = append(tlsConfig.NextProtos, http2.NextProtoTLS)
```

Comment:
I would use `tlsConfig.NextProtos = append(tlsConfig.NextProtos, []string{"h2"},` to avoid the extra import in main package.